lib/active_scheduler/resque_wrapper.rb

```diff
@@ -23,7 +25,7 @@ def self.wrap(schedule)
           class:        'ActiveScheduler::ResqueWrapper',
           queue:        queue,
           args: [{
-            job_class:  opts[:class],
+            job_class:  opts[:class] || maybe_class_name,
```

Comment:
Yes, you are correct.  Just an oversight.  Close this PR and I'll create a new one?